### PR TITLE
Adjust default figsize to match O'Reilly's requirements

### PR DESCRIPTION
**Pull request by jni**
This is a bit gnarlier than originally expected, but made good progress so thought I'd get this PR started. Having said that, for some of the squishier figures, it requires a new style file. I kinda like demoing style contexts, but it adds a tiny bit of complexity. Thoughts welcome!

---
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/302)

### CODE REVIEW

File: markdown/ch6.markdown [hunk 1]
```diff
@@ -497,39 +497,41 @@ def plot_connectome(x_coords, y_coords, conn_matrix, *,
...
 
-    fig, ax = plt.subplots(**figkwargs)
+    fig, ax = plt.subplots()
 
     # plot neuron locations:
     for neuron_type in range(ntypes):
         plotting = (types == neuron_type)
         pts = ax.scatter(x_coords[plotting], y_coords[plotting],
-                         c=cmap(neuron_type), edgecolors='face', zorder=1)
+                         c=cmap(neuron_type), s=4, zorder=1)
```
Comment (stefanv) [inline markdown/ch6.markdown]: Does s have another alias, like size?

Comment (jni) [inline markdown/ch6.markdown]: Nope! Are you amazed? Me too.

Comment (jni) [inline markdown/ch6.markdown]: (As far as I can tell anyway. Have tried size and markersize.)

**Comment by jni** [comment]
Actually this should be ready to go! Verifying local build but I think all the figsize issues should be resolved now.

**Comment by jni** [comment]
Ok build error should be fixed! Please merge at our earliest convenience and push to ATLAS! =)

File: markdown/ch6.markdown [hunk 2]
```diff
@@ -497,39 +497,41 @@ def plot_connectome(x_coords, y_coords, conn_matrix, *,
...
+                verticalalignment='center', fontsize=3, zorder=2)
 
     # plot edges
     pre, post = np.nonzero(conn_matrix)
     links = np.array([[x_coords[pre], x_coords[post]],
                       [y_coords[pre], y_coords[post]]]).T
-    ax.add_collection(LineCollection(links, color='silver',
-                                     lw=2, alpha=0.5, zorder=0))
+    ax.add_collection(LineCollection(links, color='lightgray',
+                                     lw=0.3, alpha=0.5, zorder=0))
```
Comment (stefanv) [inline markdown/ch6.markdown]: linewidths?

File: script/sliding_window.py [hunk 3]
```diff
@@ -59,4 +63,4 @@
 
 plt.savefig(fn, dpi=300)
 
-#plt.show()
+plt.show()
```
Comment (stefanv) [inline script/sliding_window.py]: Unless we set the backend to template or Agg during build, this will block the Makefile execution.

**Comment by stefanv** [comment]
Merged in https://github.com/elegant-scipy/elegant-scipy/commit/fed0f2ccdfd8a1fcbc6709e16c5f373508475632

**Pull request closed without merging**